### ISSUE TITLE
triplet-feel doesn't appear to be supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [raynebc/editor-on-fire#282]
```
What steps will reproduce the problem?
1. Create a Guitar Pro score with "triplet feel" enabled
2. Enter notes w/ the given value of the left side of the triplet feel
3. Import GP track into EOF
4. Notes will appear evenly spaced instead of with triplet feel

What is the expected output? What do you see instead?
GP score looks like attachment 1, & is interpreted as attachment 2. It is 
actually the notational equivalent of #3, & should look like #4 when imported.

What version of the product are you using? On what operating system?
Mac 1.8RC8.2

Please provide any additional information below.
There are several values for the 'triplet feel' option in GP, see #5.

```

Original issue reported on code.google.com by `ea.th1rtyf0ur` on 17 Feb 2014 at 3:19

Attachments:
* [Screen Shot 2014-02-17 at 11.58.13 PM.png](https://storage.googleapis.com/google-code-attachments/editor-on-fire/issue-282/comment-0/Screen Shot 2014-02-17 at 11.58.13 PM.png)
* [Screen Shot 2014-02-17 at 11.56.56 PM.png](https://storage.googleapis.com/google-code-attachments/editor-on-fire/issue-282/comment-0/Screen Shot 2014-02-17 at 11.56.56 PM.png)
* [Screen Shot 2014-02-17 at 11.59.14 PM.png](https://storage.googleapis.com/google-code-attachments/editor-on-fire/issue-282/comment-0/Screen Shot 2014-02-17 at 11.59.14 PM.png)
* [Screen Shot 2014-02-18 at 12.11.39 AM.png](https://storage.googleapis.com/google-code-attachments/editor-on-fire/issue-282/comment-0/Screen Shot 2014-02-18 at 12.11.39 AM.png)
* [Screen Shot 2014-02-18 at 12.17.11 AM.png](https://storage.googleapis.com/google-code-attachments/editor-on-fire/issue-282/comment-0/Screen Shot 2014-02-18 at 12.17.11 AM.png)

**Comment by th1rtyf0ur** [comment]
I found an inconsistency when starting a new project today. I've got swing/triplet feel enabled for this song:
![swing feel](https://i.imgur.com/cns9rxr.png)
![guitar pro notation](https://i.imgur.com/jH8B4iD.png)

EOF imports all the eighth notes correctly, but the dotted-quarter + eighth imports in non-swing time (the orange 2 before measure 89)

![EOF imported](https://i.imgur.com/DorDeuQ.png)

It should look like this:
![EOF corrected](http://i.imgur.com/0FeOO0L.png)

I tried googling to see if only pairs of eighth notes should be 'swung', while dotted quarter + eighth pairs should be played normally, but couldn't find anything. I would imagine the dotted quarter should behave as it generally does, i.e. like a quarter note tied to an eighth, which would 'swing' with the following eighth note.

**Comment by raynebc** [comment]
If I'm understanding it correctly, the only reason those two notes should be affected by triplet feel is that by making the quarter note dotted, it's the same as putting a tied eighth note next to another eighth note?  Are there any other special cases where EOF doesn't have triplet feel come into play?

**Comment by raynebc** [comment]
Trying this tagging thing:  @th1rtyf0ur

**Comment by th1rtyf0ur** [comment]
I haven't noticed any other issues, but I can't think of any other possibility where this would be the case, as triplet feel only affects 8th notes. Unless you have a double-dotted half note followed by an eighth note, which should behave like 3 quarter notes + an eighth note tied together. But I don't think GP5 supports double-dots, only GP6.

**Comment by raynebc** [comment]
Fixed in commit 07e56af56ea8f260e041b2ab7b0131412ba21bea
